docs/content/release_notes/csi_rn_limitations.md

```diff
@@ -41,4 +47,14 @@ When a role switch is conducted, this is not reflected within the other orchestr
  2. Delete the relevant PersistentVolumes.
  3. Import the volumes, by using the latest import procedure (version 1.7.0 or later) (see **CSI driver configuration** > **Advanced configuration** > **Importing an existing volume** in the user information).
       
-    For more information, see the [Change the Reclaim Policy of a PersistentVolume](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/) information in the Kubernetes documentation.
\ No newline at end of file
+    For more information, see the [Change the Reclaim Policy of a PersistentVolume](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/) information in the Kubernetes documentation.
+
+## HyperSwap usage limitations
+
+**Important:** The HyperSwap feature is only supported for use with IBM Spectrum Virtualize Family storage systems.
+
+The following IBM block storage CSI driver features are not supported on the volumes where HyperSwap is being used:
+
+- Volume snapshots
+- Volume cloning
+- Volume expansion
```

Comment:
update: we are working on allowing snapshots and expansion with hyperswap for 1.8.0.
so only cloning won't be supported with hyperswap

Reply:
thanks. if this is all that is left for this PR (once [this comment](https://github.com/IBM/ibm-block-csi-driver/pull/397#discussion_r742764487) is resolved) i can open another ticket and PR for the limitation changes, if needed.
or, if you prefer, I can remove those two limitations and then just add them back in later if needed.

Reply:
@oriyarde  reminder. thanks!

Reply:
let's remove those (non-cloning) limitations for now

Reply:
DONE